### ISSUE TITLE
Implement Advanced Reports via Web card

### ISSUE THREAD

**Issue by arturs105** [toggl/mobileapp#7518]
## 🎯 What needs to be done?
To increase awareness of other Toggl clients we want to indicate that the web app has better reports. To do that we need to add another card in the reports, which directs the user to the [Your Plan](https://github.com/toggl/mobileapp/issues/7516) page in the settings.

## 🎨 Graphical resources
https://toggl.invisionapp.com/share/QAXOCRTRCMY#/screens/421391237

## 🔗 Dependencies
Depends on #7516

## 🏆 Outcome
- [ ] The Advanced Reports via Web card is added
- [ ] Tapping on the card brings the user to the Your Plan page in the settings
